wrappers/python/python-rs400-advanced-mode-example.py

```diff
@@ -0,0 +1,88 @@
+## License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Please add link to the demo under [wrappers/python](https://github.com/IntelRealSense/librealsense/tree/development/wrappers/python#examples)